spec/rubocop/cop/rails/enum_hash_spec.rb

```diff
@@ -1,162 +1,375 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Rails::EnumHash, :config do
-  context 'when array syntax is used' do
-    context 'with %i[] syntax' do
-      it 'registers an offense' do
+  context 'when Rails 7 syntax is used', :rails70 do
```

Comment:
Can you add a case that ensures offenses are not detected when using `:rails61` context while still maintaining this `:rails70` context? A representative test case could used `expect_no_offenses` for the example code from L5 to L17.

e.g.,

```ruby
  context 'when Rails 6.1 or lower', :rails61 do
    context 'when array syntax is used' do
      context 'with %i[] syntax' do
        it 'does not register an offense' do
          expect_no_offenses(<<~RUBY)
            enum :status, %i[active archived]
          RUBY
        end
      end
    end
  end
```